payloads/external/SeaBIOS/Makefile

```diff
@@ -70,24 +69,11 @@ else
 	false
 endif
 endif
-#
-# Project specific configuration for optimized SeaBIOS
-#
-	echo "# CONFIG_BOOTSPLASH is not set" >> seabios/.config
-	echo "# CONFIG_MEGASAS is not set" >> seabios/.config
-	echo "# CONFIG_FLOPPY is not set" >> seabios/.config
-	echo "# CONFIG_PS2PORT is not set" >> seabios/.config
-	echo "# CONFIG_USB_UHCI is not set" >> seabios/.config
-	echo "# CONFIG_USB_OHCI is not set" >> seabios/.config
-	echo "# CONFIG_LPT is not set" >> seabios/.config
-#
-# Enable UDMA to speed up booting
-#
-	echo "CONFIG_ATA_DMA=y" >> seabios/.config
-	echo "CONFIG_ATA_PIO32=y" >> seabios/.config
-
-	# This shows how to force a previously set .config option *off*
-	#echo "# CONFIG_SMBIOS is not set" >> seabios/.config
+ifneq ($(CONFIG_SEABIOS_DEBUG_LEVEL),-1)
+	echo "CONFIG_DEBUG_LEVEL=$(CONFIG_SEABIOS_DEBUG_LEVEL)" >> seabios/.config
+endif
+#	This shows how to force a previously set .config option *off*
+#	echo "# CONFIG_SMBIOS is not set" >> seabios/.config
```

Comment:
@miczyg1 change this file really minimize difference with mainline

Reply:
@pietrushnic at this point the only difference is repo link